examples/turf.js

```diff
@@ -26,9 +26,9 @@ fetch('data/geojson/roads-seoul.geojson')
     const distance = 0.2;
 
     // get the line length in kilometers
-    const lineLength = length(turfLine, 'kilometers');
+    const lineLength = length(turfLine, {units: 'kilometers'});
     for (let i = 1; i <= lineLength / distance; i++) {
-      const turfPoint = along(turfLine, i * distance, 'kilometers');
```

Comment:
Has to be an object as seacond parameter, it worked because `'kilometers'` is the default.